runtime/tests/interpreter/interpreter_test.go

```diff
@@ -9572,6 +9572,79 @@ func TestInterpretInternalAssignment(t *testing.T) {
 	)
 }
 
+func TestInterpretVoidReturn(t *testing.T) {
```

Comment:
Good idea to generate lots of test cases, but maybe run the test cases as individual subtests. That allows them to be be debugged and resolved more easily in case of a failure